src/server/js/resources/houses/model.js

```diff
@@ -58,6 +59,8 @@ const endpoints = {
             }
         });
 
+        house.description = fields.description;
```

Comment:
we should allow partial updates. here, not specifying `description` in the update fields would set it to null.